src/lib/ScannerRunCommand.ts

```diff
@@ -126,7 +126,7 @@ export abstract class ScannerRunCommand extends ScannerCommand {
 			throw new SfdxError(messages.getMessage('validations.outfileMustBeValid'), null, null, INTERNAL_ERROR_CODE);
 		} else {
 			// Look at the file extension, and infer a corresponding output format.
-			const fileExtension = outfile.slice(lastPeriod + 1);
+			const fileExtension = outfile.slice(lastPeriod + 1).toLowerCase();
```

Comment:
@gkjung Change looks good. We may need the same fix for `--format` as well:

```
✗ sfdx scanner:run --target ~/Development/Demo/DemoApp/src --format XML
ERROR running scanner:run:  Expected --format=XML to be one of: csv, html, json, junit, sarif, table, xml
```

Reply:
As discussed, this will be addressed when a user actually feels this is an issue.